WebService/v1/Models/SimulationApiModel/SimulationRateLimits.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Concurrency;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    public class SimulationRateLimits
+    {
+        [JsonProperty(PropertyName = "ConnectionsPerSecond")]
+        public int ConnectionsPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "RegistryOperationsPerMinute")]
+        public int RegistryOperationsPerMinute { get; set; }
+
+        [JsonProperty(PropertyName = "TwinReadsPerSecond")]
+        public int TwinReadsPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "TwinWritesPerSecond")]
+        public int TwinWritesPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "DeviceMessagesPerSecond")]
+        public int DeviceMessagesPerSecond { get; set; }
+
+        // Default constructor used by web service requests
+        public SimulationRateLimits()
+        {
+            this.ConnectionsPerSecond = 0;
+            this.RegistryOperationsPerMinute = 0;
+            this.TwinReadsPerSecond = 0;
+            this.TwinWritesPerSecond = 0;
+            this.DeviceMessagesPerSecond = 0;
+        }
+
+        // Map API model to service model
+        public static SimulationRateLimits FromServiceModel(
+            Services.Models.Simulation.SimulationRateLimits simulationRateLimits,
+            IRateLimiting rateReporter)
+        {
+            var defaultRatingLimits = rateReporter.GetRateLimits();
```

Comment:
I would add unit tests to make sure that the defaults are used

Reply:
added